libdnf/dnf-context.cpp

```diff
@@ -1191,6 +1210,22 @@ dnf_context_set_zchunk(DnfContext *context, gboolean zchunk)
     priv->zchunk = zchunk;
 }
 
+/**
+ * dnf_context_set_write_history:
+ * @context: a #DnfContext instance.
+ * @value: %TRUE write history database
+ *
+ * Enables or disables writing to history database.
+ *
+ * Since: 0.25.0
```

Comment:
Just bump this version to match what it should be...

Reply:
fixed